pkgs/development/tools/misc/ast-grep/default.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
```

Comment:
Isn't the nixpkgs style to have commas at the beginning?

Reply:
Yes you are right I have overlooked that.

Reply:
Reformatted with `nixpkgs-fmt`